packages/core/src/external/athenahealth/index.ts

```diff
@@ -51,18 +60,49 @@ function getS3UtilsInstance(): S3Utils {
   return new S3Utils(region);
 }
 
+const problemStatusesAthena = ["CHRONIC", "ACUTE"];
+// TODO import from shared?
```

Comment:
left-over? or intentionally left for later (in which case, I'd suggest adding a ticket number to it)?

Reply:
Removed.